Source/FolioReaderPage.swift

```diff
@@ -186,129 +186,156 @@ open class FolioReaderPage: UICollectionViewCell, UIWebViewDelegate, UIGestureRe
         }
         return tempHtmlContent as String
     }
-
-    // MARK: - UIWebView Delegate
-
-    open func webViewDidFinishLoad(_ webView: UIWebView) {
+    
+    // MARK: - WKNavigationDelegate
+    
+    public func webView(_ webView: WKWebView, didCommit navigation: WKNavigation!) {
```

Comment:
This was `open` so anyone can override it if they want.
```suggestion
    open func webView(_ webView: WKWebView, didCommit navigation: WKNavigation!) {
```